### ISSUE TITLE
[pixelhd] You appear to have cookies disabled. (Config)

### ISSUE THREAD

**Issue by scooterh928** [Jackett/Jackett#2363]
Not getting the captcha image, shows like a image should be there but not loading, and when i ignore the captcha text i get the cookies are disabled, and i've checked and they are enabled, have several other private trackers in Jackett with no issues, so far only PixelHD one i've had issues with.

Log Error:

Jan 03, 2018 08:36:31 PM | Error | {0}
-- | -- | --

iMac:~ scooter$ mono -V
Mono JIT compiler version 4.8.1 (mono-4.8.0-branch/22a39d7 Fri Apr  7 12:00:08 EDT 2017)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           normal
	SIGSEGV:       altstack
	Notification:  kqueue
	Architecture:  x86
	Disabled:      none
	Misc:          softdebug 
	LLVM:          yes(3.6.0svn-mono-master/8b1520c)
	GC:            sgen

Jackett Version 0.8.562.0

**Comment by kaso17** [comment]
Can't reproduce it.
Captcha seems to be disabled at this time, so ignoring it is fine.
The cookie error was a false positive (ignore it), there must be something else wrong.
Please provide the log.txt file from the jackett directory.

**Comment by scooterh928** [comment]
Jan 05, 2018 06:31:03 PM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Logs()'
-- | -- | --
Jan 05, 2018 06:31:03 PM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Logs()'
Jan 05, 2018 06:31:03 PM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
Jan 05, 2018 06:31:03 PM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Logs()'
Jan 05, 2018 06:31:03 PM | Info | HttpActionBinding ExecuteBindingAsync
Jan 05, 2018 06:31:03 PM | Info | HttpActionBinding ExecuteBindingAsync
Jan 05, 2018 06:31:03 PM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Jan 05, 2018 06:31:03 PM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Jan 05, 2018 06:31:03 PM | Info | ApiControllerActionSelector SelectAction Selected action 'Logs()'
Jan 05, 2018 06:31:03 PM | Info | ApiControllerActionSelector SelectAction
Jan 05, 2018 06:31:03 PM | Info | ServerConfigurationController ExecuteAsync
Jan 05, 2018 06:31:03 PM | Info | HttpControllerDescriptor CreateController Jackett.Controllers.V20.ServerConfigurationController
Jan 05, 2018 06:31:03 PM | Info | DefaultHttpControllerActivator Create Jackett.Controllers.V20.ServerConfigurationController
Jan 05, 2018 06:31:03 PM | Info | DefaultHttpControllerActivator Create
Jan 05, 2018 06:31:03 PM | Info | HttpControllerDescriptor CreateController
Jan 05, 2018 06:31:03 PM | Info | DefaultHttpControllerSelector SelectController ServerConfiguration
Jan 05, 2018 06:31:03 PM | Info | DefaultHttpControllerSelector SelectController Route='action:logs,controller:ServerConfiguration'
Jan 05, 2018 06:31:03 PM | Info | http://127.0.0.1:9117/api/v2.0/server/logs?_=1515198623846
Jan 05, 2018 06:30:56 PM | Info | IndexerApiController Dispose
Jan 05, 2018 06:30:56 PM | Info | IndexerApiController Dispose
Jan 05, 2018 06:30:56 PM | Info | Content-type='application/json; charset=utf-8', content-length=23455
Jan 05, 2018 06:30:56 PM | Info | IndexerApiController ExecuteAsync
Jan 05, 2018 06:30:56 PM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
Jan 05, 2018 06:30:56 PM | Error | {0}
Jan 05, 2018 06:30:56 PM | Info | ApiExceptionHandler ExecuteExceptionFilterAsync
Jan 05, 2018 06:30:56 PM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync
Jan 05, 2018 06:30:56 PM | Info | JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
Jan 05, 2018 06:30:56 PM | Info | RequiresIndexerAttribute OnActionExecutedAsync
Jan 05, 2018 06:30:56 PM | Info | RequiresIndexerAttribute OnActionExecutedAsync Action filter for 'Config(ConfigItem[] config)'
Jan 05, 2018 06:30:56 PM | Error | ApiControllerActionInvoker InvokeActionAsync
Jan 05, 2018 06:30:56 PM | Error | ReflectedHttpActionDescriptor ExecuteAsync
Jan 05, 2018 06:30:56 PM | Debug | WebClient(HttpWebClient): Returning OK => <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"> <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"> <head> <title>Login :: PiXELHD</title> <meta http-equiv="X-UA-Compatible" content="chrome=1;IE=edge" /> <meta http-equiv="Content-Type" content="text/html; charset=utf-8" /> <meta name="globalsign-domain-verification" content="qR3XYrQm_cLObph35regEvEhufRzOEB26lz6cTz2gE" /> <link rel="shortcut icon" href="favicon.ico" /> <link rel="apple-touch-icon-precomposed" sizes="512x512" href="static/pxhd_icon.png" /> <link href="static/styles/public/style.css?v=1449239447" rel="stylesheet" type="text/css" /> <script src="static/functions/script_start.js?v=1449239394" type="text/javascript"></script> <script src="static/functions/class_ajax.js?v=1449239393" type="text/javascript" async="async"></script> <script src="static/functions/class_cookie.js?v=1449239393" type="text/javascript" async="async"></script> <script src="static/functions/class_storage.js?v=1449239393" type="text/javascript"></script> <script src="static/functions/global.js?v=1449239393" type="text/javascript"></script> <script src="static/functions/jquery.js" type="text/javascript"></script> <script src="static/functions/jquery-ui-1.10.2.custom.min.js" type="text/javascript"></script> </head> <body> <div id="head"> </div> <table id="maincontent"> <tr> <td align="center" valign="middle"> <div id="logo"> <ul> <li><a href="index.php">Home</a></li> <li><a href="login.php">Login</a></li> <li><a href="apply.php">Apply</a></li> </div> <span id="no-cookies" class="hidden warning">You appear to have cookies disabled.<br /><br /></span> <noscript><span class="warning">You appear to have javascript disabled.</span><br/><br/></noscript> <form class="fade_in" id="eff43eb8f1f21b5b0cf32dc2dcb54f9bfe397646" method="post" action="login.php"> <span class="warning">Incorrect username or password.<br /><br /></span> You have <span class="info">4</span> attempts remaining.<br /><br /> <strong>WARNING:</strong> You will be banned for 6 hours after your login attempts run out!<br /><br /> <table> <tr> <td>&nbsp;</td> <td colspan="2"><input name="72d3c4a5f8fee6352c4c4d84a5414bb5325e09a5" id="72d3c4a5f8fee6352c4c4d84a5414bb5325e09a5" class="emanresu" required="required" maxlength="20" pattern="[A-Za-z0-9_?]{1,20}" autofocus="autofocus" /></td> </tr> <tr> <td>&nbsp;</td> <td colspan="2"><input name="eae0841e10a556550516cc592f56b2972e465084" id="eae0841e10a556550516cc592f56b2972e465084" class="drowssap" required="required" type="password" maxlength="40" pattern=".{6,40}" /></td> </tr> <tr> <td></td> <td> <input type="checkbox" id="keeplogged" name="keeplogged" value="1" checked="checked" /> <label for="keeplogged">Remember me</label> </td> <td><input type="submit" id="d9ca8e9d1dff473825ed9bb0bb47a476fe467f90" name="d9ca8e9d1dff473825ed9bb0bb47a476fe467f90" value="Login" class="submit" /></td> </tr> </table> <br /> <div style="text-align: center;"> <img id="6773165b5e685607a34c4361f21fde1cf9d4ad22" src="/fuckyou/securimage_show.php" alt="FuckOff Image" /><br /> <input type="text" name="f49831e23b6ad1e348c5d59e2ed1c1a1fe34b06a" size="10" maxlength="6" /><br /> [<a href="#" onclick="document.getElementById('6773165b5e685607a34c4361f21fde1cf9d4ad22').src = '/fuckyou/securimage_show.php?' + Math.random(); return false">Different Image</a>] </div> </form> <script type="text/javascript">         $ ( document ).ready ( function () {             $ ( "p.fade_1" ).fadeIn ( 2000 );         } );     </script> <div style="margin: 20px;"> <table> <tr> <td width="500px" style="text-align: center;"> <p class="fade_1" style="display:none;">Mama always said life was like a box of chocolates. You never know what you&#39;re gonna get.</p> </td> </tr> </table> </div> <br /><br /> Lost your password? <a href="login.php?act=recover">Recover it here!</a> <script type="text/javascript">     cookie.set ( 'cookie_test', 1, 1 );     if ( cookie.get ( 'cookie_test' ) != null ) {         cookie.del ( 'cookie_test' );     } else {         $ ( '#no-cookies' ).show ();     } </script> </td> </tr> </table> <div id="foot"> <span> <a href="https://pixelhd.me/#">PiXELHD</a>&nbsp;&nbsp; \|&nbsp;&nbsp; <a href="https://img.pixelhd.me">IMG.PiXELHD</a>&nbsp;&nbsp; \|&nbsp;&nbsp; subLiME </span> </div> </body> </html>
Jan 05, 2018 06:30:55 PM | Debug | WebClient(HttpWebClient).GetString(Url:https://pixelhd.me/login.php)
Jan 05, 2018 06:30:55 PM | Info | ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Config(config=Jackett.Models.DTO.ConfigItem[])'
Jan 05, 2018 06:30:55 PM | Info | ApiControllerActionInvoker InvokeActionAsync Action='Config(config=Jackett.Models.DTO.ConfigItem[])'
Jan 05, 2018 06:30:55 PM | Info | RequiresIndexerAttribute OnActionExecutingAsync
Jan 05, 2018 06:30:55 PM | Info | RequiresIndexerAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
Jan 05, 2018 06:30:55 PM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync
Jan 05, 2018 06:30:55 PM | Info | JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Config(ConfigItem[] config)'
Jan 05, 2018 06:30:55 PM | Info | HttpActionBinding ExecuteBindingAsync Model state is valid. Values: config=Jackett.Models.DTO.ConfigItem[]
Jan 05, 2018 06:30:55 PM | Info | FormatterParameterBinding ExecuteBindingAsync Parameter 'config' bound to the value 'Jackett.Models.DTO.ConfigItem[]'
Jan 05, 2018 06:30:55 PM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Value read='Jackett.Models.DTO.ConfigItem[]'
Jan 05, 2018 06:30:55 PM | Info | JsonMediaTypeFormatter ReadFromStreamAsync Type='ConfigItem[]', content-type='application/json'
Jan 05, 2018 06:30:55 PM | Info | FormatterParameterBinding ExecuteBindingAsync Binding parameter 'config'
Jan 05, 2018 06:30:55 PM | Info | HttpActionBinding ExecuteBindingAsync
Jan 05, 2018 06:30:55 PM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Jan 05, 2018 06:30:55 PM | Info | JackettAuthorizedAttribute OnAuthorizationAsync
Jan 05, 2018 06:30:55 PM | Info | ApiControllerActionSelector SelectAction Selected action 'Config(ConfigItem[] config)'
Jan 05, 2018 06:30:55 PM | Info | ApiControllerActionSelector SelectAction
Jan 05, 2018 06:30:55 PM | Info | IndexerApiController ExecuteAsync



[log.20180105.00000.txt](https://github.com/Jackett/Jackett/files/1608275/log.20180105.00000.txt)
[log.20180105.00001.txt](https://github.com/Jackett/Jackett/files/1608276/log.20180105.00001.txt)

**Comment by scooterh928** [comment]
[log.txt](https://github.com/Jackett/Jackett/files/1608280/log.txt)

**Comment by scooterh928** [comment]
figured it out since enabling advanced logging, guess entering wrong password :/

sorry for your trouble, and thank you for your time.

**Comment by kaso17** [comment]
Thank you for bringing it up anyway.
Login error parsing has been fixed via e92b87e4dd5b9dbfa7918c062809021bed95e3e4
